Error in AutoCreateTeams

**Describe the bug**
When ever i try to setup map and choose Auto Create Teams it send this output

**To Reproduce**
Steps to reproduce the behavior:
1. Setup a arena 
2. Click on 'AutoCreateTeams'
3. See error

[09:08:21 INFO]: Anas1554 issued server command: /bw setWaitingSpawn
[09:08:23 ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing 'bw autocreateteams' in com.andrei1058.bedwars.commands.bedwars.MainCommand(bw)
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:155) ~[patched.jar:git-PaperSpigot-"4c7641d"]
        at org.bukkit.craftbukkit.v1_8_R3.CraftServer.dispatchCommand(CraftServer.java:666) ~[patched.jar:git-PaperSpigot-"4c7641d"]
        at org.bukkit.Bukkit.dispatchCommand(Bukkit.java:596) ~[patched.jar:git-PaperSpigot-"4c7641d"]
        at com.andrei1058.bedwars.commands.bedwars.subcmds.sensitive.setup.SetWaitingSpawn.execute(SetWaitingSpawn.java:56) ~[?:?]
        at com.andrei1058.bedwars.commands.bedwars.MainCommand.execute(MainCommand.java:149) ~[?:?]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:146) ~[patched.jar:git-PaperSpigot-"4c7641d"]
        at org.bukkit.craftbukkit.v1_8_R3.CraftServer.dispatchCommand(CraftServer.java:666) ~[patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PlayerConnection.handleCommand(PlayerConnection.java:1189) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PlayerConnection.a(PlayerConnection.java:1001) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PacketPlayInChat.a(PacketPlayInChat.java:45) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PacketPlayInChat.a(PacketPlayInChat.java:1) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PlayerConnectionUtils$1.run(SourceFile:13) [patched.jar:git-PaperSpigot-"4c7641d"]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_312]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_312]
        at net.minecraft.server.v1_8_R3.SystemUtils.a(SourceFile:44) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.B(MinecraftServer.java:774) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.DedicatedServer.B(DedicatedServer.java:378) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.A(MinecraftServer.java:713) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:616) [patched.jar:git-PaperSpigot-"4c7641d"]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
Caused by: java.lang.IllegalArgumentException: No enum constant com.andrei1058.bedwars.api.arena.team.TeamColor.LGRAY
        at java.lang.Enum.valueOf(Enum.java:238) ~[?:1.8.0_312]
        at com.andrei1058.bedwars.api.arena.team.TeamColor.valueOf(TeamColor.java:29) ~[?:?]
        at com.andrei1058.bedwars.api.arena.team.TeamColor.getChatColor(TeamColor.java:50) ~[?:?]
        at com.andrei1058.bedwars.commands.bedwars.subcmds.sensitive.setup.AutoCreateTeams.execute(AutoCreateTeams.java:212) ~[?:?]
        at com.andrei1058.bedwars.commands.bedwars.MainCommand.execute(MainCommand.java:149) ~[?:?]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:146) ~[patched.jar:git-PaperSpigot-"4c7641d"]
        ... 19 more

**Screenshots**
[Screenshots](https://i.imgur.com/rD1ZVPy.png)

**System (please complete the following information):**
 - OS: [Ubuntu 20.04]
 - Server software [Paper]
 - Server Version [1.8.8]
 - Plugin Version [21.10.1]

--- a/bedwars-api/src/main/java/com/andrei1058/bedwars/api/arena/team/TeamColor.java
+++ b/bedwars-api/src/main/java/com/andrei1058/bedwars/api/arena/team/TeamColor.java
@@ -289,10 +289,10 @@ public enum TeamColor {
                 name = "White";
                 break;
             case 8:
-                name = "Gray";
+                name = "Dark_Gray";
                 break;
             case 7:
-                name = "lGray";
+                name = "Gray";
                 break;
 
         }

--- a/bedwars-plugin/src/main/java/com/andrei1058/bedwars/commands/bedwars/subcmds/sensitive/setup/AutoCreateTeams.java
+++ b/bedwars-plugin/src/main/java/com/andrei1058/bedwars/commands/bedwars/subcmds/sensitive/setup/AutoCreateTeams.java
@@ -209,7 +209,7 @@ public class AutoCreateTeams extends SubCommand {
                     p.sendMessage("§6§lNEW TEAMS FOUND:");
                     for (Byte tf : found) {
                         String name = TeamColor.enName(tf);
-                        p.sendMessage("§f ▪ " + TeamColor.getChatColor(name) + name);
+                        p.sendMessage("§f ▪ " + TeamColor.getChatColor(name) + name.replace('_', ' '));
                     }
                     p.spigot().sendMessage(Misc.msgHoverClick("§6 ▪ §7§lClick here to create found teams.", "§fClick to create found teams!", "/" + getParent().getName() + " " + getSubCommandName(), ClickEvent.Action.RUN_COMMAND));
                 }

